### ISSUE TITLE
508-defect-2 [HEADINGS, AXE-CORE]: Apply page, heading levels should only increase by one

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#22845]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## Details

The "Save time-- and save your progress" box is a `h3` which breaks the heading orders on the page. It appears the "Follow the steps" section's headings were all pushed down a heading level to accommodate this rule. Unfortunately, this only makes the organization of the page even more confusing.

Heading levels should be thought of as an outline of a page. Currently, given the order, the "follow the steps" section are directly nested below the "Save time-- and save your work" alert which I don't believe was the intent of the design.

## Acceptance Criteria

- [ ] Heading levels are not skipped on the page

## Steps to Recreate

1. Enter `https://staging.va.gov/careers-employment/vocational-rehabilitation/apply/introduction` in browser
2. [Use headingsMap to verify this is an issue](https://chrome.google.com/webstore/detail/headingsmap/flbjommegcjonpdmenkdiocclhjacmbi?hl=en) or run an axe-scan
3. Confirm headings are skipped

## Proposed Solution 

Instead of the following order, which doesn't make semantic sense as an outline...

1. `h1` Apply for Veteran Readiness...
  1. [h2 is missing] 
    1. `h3` Save time--and save your work..
      1. `h4` Follow the steps below...
        1. `h5` Prepare
          1. `h6` To fill out this...
        2. `h5` Apply
        3. `h5` VA Review
        4. `h5` Decision

We should rearrange the order like this:

1. `h1` Apply for Veteran Readiness...
  1. `h2` Save time--and save your work..
  2. `h2` Follow the steps below...
    1. `h3` Prepare
      1. `h4` To fill out this...
    2. `h3` Apply
    3. `h3` VA Review
    4. `h3` Decision

Should we be unable to adjust the sign in's `h3`, keep this ticket open, but make the following changes:

1. `h1` Apply for Veteran Readiness...
    1. `h3` Save time--and save your work..
  2. `h2` Follow the steps below...
    1. `h3` Prepare
      1. `h4` To fill out this...
    2. `h3` Apply
    3. `h3` VA Review
    4. `h3` Decision

It's better the majority of the page's structure makes sense rather than _all_ of the page making no sense.

## WCAG or Vendor Guidance (optional)

* [Heading order, AXE, deque](https://dequeuniversity.com/rules/axe/4.1/heading-order?application=coconut)

## Screenshots or Trace Logs

<img width="1792" alt="Screen Shot 2021-04-07 at 12 03 52 PM" src="https://user-images.githubusercontent.com/14154792/113899325-b0afc880-979a-11eb-9774-898cd999dfdf.png">

**Comment by micahchiang** [comment]
@jason-gcio @joshkimux - I think this can be closed. Axe no longer shows a heading level issue and the mentioned heading has been fixed:
![Screen Shot 2021-09-29 at 2.48.47 PM.png](https://images.zenhubusercontent.com/5d68043b330f64311395e92b/f0860a6e-a759-44c5-94bc-2740d67ff803)
![Screen Shot 2021-09-29 at 2.50.42 PM.png](https://images.zenhubusercontent.com/5d68043b330f64311395e92b/2bc02e62-7935-4702-9f31-c4dc40805b03)

**Comment by joshkimux** [comment]
Closing this! Thanks @micahchiang
